wdl2cwl/tests/cwl_files/align_and_count.cwl

```diff
@@ -18,16 +18,16 @@ inputs:
     default: quay.io/broadinstitute/viral-core:2.1.33
     type: string
 outputs:
-  - id: report
+  - id: reports.align_and_count.report
     outputSource: reports.align_and_count/report
     type: File
-  - id: report_top_hits
+  - id: reports.align_and_count.report_top_hits
     outputSource: reports.align_and_count/report_top_hits
     type: File
-  - id: top_hit_id
+  - id: reports.align_and_count.top_hit_id
     outputSource: reports.align_and_count/top_hit_id
     type: string
-  - id: viralngs_version
+  - id: reports.align_and_count.viralngs_version
     outputSource: reports.align_and_count/viralngs_version
     type: string
```

Comment:
Yeah, I disagree with this. 

The original WDL file has explicit outputs, without namespaces. That shouldn't change. https://github.com/common-workflow-lab/wdl-cwl-translator/blob/4d87965faa68fd52a400ff20db84814d4b9ffdb2/wdl2cwl/tests/wdl_files/align_and_count.wdl#L16